atlascope/core/models/dataset.py

```diff
@@ -1,8 +1,15 @@
+import os
+from pathlib import Path
+import tempfile
+
 from django.conf import settings
 from django.contrib import admin
 from django.core.exceptions import ValidationError
 from django.db import models
+from django.dispatch import receiver
 from django_extensions.db.models import TimeStampedModel
+from large_image_converter import _output_tiff
```

Comment:
I'm a little nervous about importing a "private" (prefixed by underscore) method. We should definitely pin the version of `large_image_converter` if we do this. Otherwise, is there an official public method we can use?

Reply:
this private function was all that I could find to do what we need. Some public functions invoke this but do much more additional stuff. The module as a whole has most of its functions as private with very narrow public use cases. I pinned the working version of `large_image_converter`, though. For now that will have to do. [24b2e61](https://github.com/atlascope/atlascope/pull/140/commits/24b2e610665820fb01468468b58239c2ebdaf749)

Reply:
@manthey, can you comment on this? Could it be possible to make `_output_tiff()` an official part of the API?

Reply:
It looks like you are really trying to do what `tifftools.commands.tiff_concat` does.  I haven't digested what you are doing besides concatenating a set of tiled tiff images.

Reply:
I opened https://github.com/atlascope/atlascope/pull/166 to use tiff_concat instead. It works well!